clients/python/lakefs_client/client.py

```diff
@@ -1,9 +1,18 @@
+from urllib.parse import urlparse, urlunparse
+
 import lakefs_client
-from lakefs_client.apis import ActionsApi, AuthApi, BranchesApi, CommitsApi, ConfigApi, HealthCheckApi, MetadataApi, ObjectsApi, RefsApi, RepositoriesApi, StagingApi, TagsApi
+from lakefs_client.apis import ActionsApi, AuthApi, BranchesApi, CommitsApi, ConfigApi, HealthCheckApi, MetadataApi, \
+    ObjectsApi, RefsApi, RepositoriesApi, StagingApi, TagsApi
+
+DEFAULT_BASE_URL_PATH = "/api/v1"
```

Comment:
Doesn't exist in the generated code?
If it does use it instead of copying it here.

Reply:
added code to extract the path from the first host configuration